packages/variable-editor/src/components/variables/master/AddVariableDialog.css

```diff
@@ -1,3 +1,11 @@
 .add-variable-dialog-trigger-button {
   margin-left: auto;
 }
+
+.ui-combobox-menu {
+  z-index: 50;
+}
+
+.ui-combobox-menu:empty {
+  display: none;
+}
```

Comment:
needed?

Reply:
Not necessarily, but I think it works better. If I remove it, things like this seem cumbersome:
![hide-empty-menu](https://github.com/axonivy/config-editor-client/assets/141232142/c54044ec-697a-4181-ac5c-01fd6da7ccf6)